src/sql/workbench/parts/dashboard/widgets/properties/propertiesWidget.component.html

```diff
@@ -8,13 +8,19 @@
 	<div [style.margin-right.px]="_clipped ? 30 : 0" [style.width]="_clipped ? 94 + '%' : '100%'" style="overflow: hidden">
 		<span #child style="white-space : nowrap; width: fit-content">
 			<ng-template ngFor let-item [ngForOf]="properties">
-				<span style="margin-left: 10px; display: inline-block;">
+				<span *ngIf="!_isVertical" style="margin-left: 10px; display: inline-block;">
 					<div style="font-size: 11px;">{{item.displayName}}</div>
 					<div>{{item.value}}</div>
 				</span>
+				<div class="flex-table-row" *ngIf="_isVertical">
```

Comment:
>flex-table-row [](start = 16, length = 14)

flex-table-row for this when _isVertical is true? Wouldn't flex-table-column make more sense?

Reply:
I added it to avoid using the models to decide the orientation of the data, which isn't extensible. And vertical is when the key value pairs are vertical one below the other. When _isVertical is false, the key value pairs are next to each to other. So changing that wouldn't make sense.

Reply:
Then I'd suggest isStackedVertical which is a bit clearer, right now what you're saying isn't clear at all.

Reply:
is isTabular better ? Since this actually makes it look like a table?